pkg/cmd/cli/cmd/status.go

```diff
@@ -22,21 +25,36 @@ about individual items, use the describe command (e.g. oc describe buildConfig,
 oc describe deploymentConfig, oc describe service).`
 
 	statusExample = `  // Show an overview of the current project
-  $ %[1]s status`
+  $ %[1]s
+
+  // Show a digraph of the current project in DOT format
```

Comment:
I don't know that we should show more than one example.  Add a paragraph describing it.  Also, I kind of think this should be "-o dot" now, just seeing the usage.